### DISCUSSION TITLE
How to update PnP to latest nightly on Powershell 5 ?

### DISCUSSION THREAD

**Discussion by craces** [pnp/powershell#1907] [Q&A]
Hi, i have an issue on latest stable version , case like this: https://github.com/pnp/powershell/issues/1618
and i want to update to latest nightly to fix it, but it looks like PS5 does not recognize -allowprerelease param.

![image](https://user-images.githubusercontent.com/26762445/170862207-ec0fb3d8-c7d9-4e32-bada-3265141f1167.png)

Is any solution for that ?

**Comment by veronicageek** [comment]
@craces - The error message is not referring to the `-AllowPrerelease`  parameter. It's about the `-RequiredVersion`. To update PnP PowerShell to the nightly release, you don't need to specify the version. Just `Install-Module pnp.powershell -AllowPrerelease -Force` is enough to get the latest.

**Comment by craces** [comment]
@veronicageek im, not sure 
![image](https://user-images.githubusercontent.com/26762445/179498015-7e2bcbb7-63fe-4cdf-9f73-68ac06b47e9d.png)

**Comment by CreativeAcer** [comment]
@craces run the following command:
`Install-Module PowershellGet -Force`

Then check new version with `Get-Command Install-Module`
Close Shell and Reopen - try again, now your prerelease parameter should work (i'm currently on Install-Module 2.2.5

**Comment by veronicageek** [comment]
@craces - PnP PowerShell 2.2.5 will not work with PS5 as we mentioned in the [README](https://github.com/pnp/powershell#important---new-pnp-powershell-2x) file.

For PS5 you need maximum 1.12.0 or 1.12.80-nightly.

`Install-Module PnP.PowerShell -RequiredVersion 1.12.0 -Force` or `Install-Module PnP.PowerShell -AllowPrerelease -RequiredVersion 1.12.80-nightly -Force`.

**Reply by CreativeAcer** [reply]
Missed the PS5 part, any reason you would stick to that version?
